### ISSUE TITLE
Running with python -m?

### ISSUE THREAD

**Issue by bbimber** [broadinstitute/CellBender#123]
Hello,

This is possibly a basic question: we'd like to call cellbender from an R package. We cannot guarantee the user has the cellbander executable in their PATH. It is generally possible to run a python package using something like:

python -m cellbender.base_cli remove-background <args>

Should this work for cellbender? In my testing this wasnt printing the help options, but my python knowledge isnt that high so perhaps I'm missing something. 

Thanks for any ideas.

**Comment by sjfleming** [comment]
Hi @bbimber ... hmm, I am not sure, I have actually never tried running a python package that way either.

Okay based on what I am seeing, cellbender is NOT currently set up to be run this way.  But there is a simple fix, which I think is compatible with everything else.

I will plan to add this code at the end of `base_cli.py`:

```python

if __name__ == '__main__':
    sys.exit(main())

```

It might take me a little while to get this merged into a new release, since there's a lot going into this next release.  But if you want to try this out yourself on your own fork, for example, you can go ahead and it should work.  It seems to work for me:

```
$ python -m cellbender.base_cli remove-background -h

usage: cellbender remove-background [-h] --input INPUT_FILE --output
...
```

**Comment by sjfleming** [comment]
Closed by #238
